experiments/ClimaEarth/Project.toml

```diff
@@ -58,9 +58,9 @@ ArgParse = "1.1"
 ArtifactWrappers = "0.2"
 AtmosphericProfilesLibrary = "0.1"
 ClimaAnalysis = "0.5.4"
-ClimaAtmos = "0.27"
+ClimaAtmos = "0.27.2"
```

Comment:
Are we using features from 0.27.2? If not, we should not force this compat and let Julia pick the most updated version.

(Same with Land below)

Reply:
I'll update, rebase and merge